src/test/java/org/cactoos/io/TempFileTest.java

```diff
@@ -92,11 +92,11 @@ public void createFileWithPrefix() throws Exception {
             System.currentTimeMillis()
         ).asString();
         try (TempFile file = new TempFile(prefix, "")) {
-            MatcherAssert.assertThat(
+            new Assertion<>(
                 "File not created with the given prefix",
                 file.value().getFileName().toString(),
                 Matchers.startsWith(prefix)
```

Comment:
@marceloamadeu can you change that to the object oriented version coming from cactoos-matchers?

Reply:
@victornoel Done ... please check.